src/components/Footer/Footer.css

```diff
@@ -1,69 +1,69 @@
-@import '../../pages/Home/Home.css';
+@import "../../pages/Home/Home.css";
 
 /*Footer Base*/
 .footer {
-    padding-top: 10px;
-    font-family: "Poppins", sans-serif;
-    background-color: #121212;
+  padding-top: 10px;
+  font-family: "Poppins", sans-serif;
+  background-color: #121212;
```

Comment:
Use variables for the colors. Name them appropriately. Declare separately for dark mode and light mode. Check the style.css in the master branch please